drwmutex.go

```diff
@@ -179,18 +171,23 @@ func lock(clnts []NetLocker, locks *[]string, lockName string, isReadLock bool)
 			if locked {
 				g.lockUid = args.UID
 			}
+
 			ch <- g
 
 		}(index, isReadLock, c)
 	}
 
 	quorum := false
 
-	var wg sync.WaitGroup
 	wg.Add(1)
 	go func(isReadLock bool) {
 
-		// Wait until we have either a) received all lock responses, b) received too many 'non-'locks for quorum to be or c) time out
+		// Wait until we have either
+		//
+		// a) received all lock responses,
+		// b) received too many 'non-'locks for quorum to be or
```

Comment:
Missing end of sentence:
- received too many 'non-'locks for quorum to be **still possible**